webapp/portlet/src/main/webapp/vue-apps/people-list/initComponents.js

```diff
@@ -4,6 +4,9 @@ import PeopleToolbar from './components/PeopleToolbar.vue';
 import PeopleCard from './components/PeopleCard.vue';
 import PeopleCardFront from './components/PeopleCardFront.vue';
 import PeopleCardReverse from './components/PeopleCardReverse.vue';
+import PeopleAdvancedFilterDrawer from './components/PeopleAdvancedFilterDrawer.vue';
+
+import * as ProfileSettingsService from '../common/js/ProfileSettingsService.js';
```

Comment:
if this is a common service that is defined in `commonVueComponent`, so no need to redefine it here.
```suggestion
```

Reply:
yes already mentioned [here](https://github.com/Meeds-io/social/pull/2191#issuecomment-1410424867), it is declared like that since in the other MIP it isn't yet a common component, so it should be updated there before.

Reply:
fixed by this commit  https://github.com/Meeds-io/social/commit/616cd490ff344bc114a19d5b9f9163c1f4bb58a5

Reply:
fixed